testsuite/integration/rts/src/test/java/org/wildfly/test/extension/rts/common/WorkRestATResource.java

```diff
@@ -32,7 +32,7 @@
 import org.jboss.jbossts.star.util.TxSupport;
 
 @Path(WorkRestATResource.PATH_SEGMENT)
-public final class WorkRestATResource {
+public class WorkRestATResource {
```

Comment:
Making this final makes the type non-proxyable.

Reply:
+1